packages/cli/src/checks/__tests__/INCORRECT_REPOSITORY_FIELD.ts

```diff
@@ -169,9 +169,9 @@ describe("incorrect repository field", () => {
       (rootWorkspace.packageJson as any).repository =
         "https://dev.azure.com/Thinkmill/monorepos/_git/manypkg";
 
-      rootWorkspace.dir = __dirname;
+      rootWorkspace.relativeDir = ".";
```

Comment:
it seemed a little bit weird that we have to mutate this stuff here, it's outside of the scope of this PR so doesn't have to be fixed here - I prepared a small PR targeting yours though: https://github.com/elliot-nelson/manypkg/pull/5